packages/utils/interactive-hooks/src/__tests__/useControllableValue.test.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+
+import { validateHookValueNotChanged } from '@fluentui-react-native/test-tools';
+import { mount } from 'enzyme';
+
+import { useControllableValue } from '../useControllableValue';
+
+describe('useControllableValue', () => {
+  it('respects controlled value', () => {
+    let resultValue: boolean | undefined;
+    const TestComponent: React.FunctionComponent<{ value?: boolean; defaultValue?: boolean }> = ({ value, defaultValue }) => {
+      [resultValue] = useControllableValue(value, defaultValue);
+      return <div />;
```

Comment:
Does <div> make sense given that this is a React Native library? I think View seems like a better fit.

Reply:
Removed. Thank you.